api/wb/views.py

```diff
@@ -24,10 +24,14 @@ def get_object(self):
         return self.get_target(self.kwargs[self.target_lookup_url_kwarg])
 
     def get_target(self, target_id):
-        guid = Guid.load(target_id)
-        if not guid:
-            raise NotFound
-        target = guid.referent
+        #TODO use Guid.load_referent when pr will merge
+        if VersionedGuidMixin.GUID_VERSION_DELIMITER in target_id:
+            target, version = Guid.load_referent(target_id)
+        else:
+            guid = Guid.load(target_id)
+            if not guid:
+                raise NotFound
+            target = guid.referent
```

Comment:
```suggestion
        target, _ = Guid.load_referent(target_id)
        if not target:
            raise NotFound
```

`.load_referent()` does the job for us